<!-- http://wiki.ros.org/roslaunch/XML/node -->
<launch>
   <node name="undistort_node" pkg="ocamcalib_undistort" type="ocamcalib_undistort_node" output="screen">

        <param type="string" name="camera_type" value="fisheye" />
        <param type="string" name="base_in_topic" value="/camera/image" />

        <param type="string" name="calibration_file_path" value="include/calib_results.txt" />
        <param type="string" name="transport_hint" value="raw" />
        <param type="double" name="scale_factor" value="4" />
        <param type="int" name="right_bound" value="640" />
        <param type="int" name="left_bound" value="0" />
        <param type="int" name="bottom_bound" value="480" />
        <param type="int" name="top_bound" value="0" />
    </node>
</launch>
